mstranslator.py

```diff
@@ -89,143 +94,109 @@ def make_response(self, resp):
         }
+
         if lang_from:
             params['from'] = lang_from
-        params.update(text_params)
 
-        return self.make_request(action, params)
+        return self.make_request(action, params, body)
 
     def translate(self, text, lang_from=None, lang_to=None,
                   contenttype='text/plain', category='general'):
-        params = {
-            'text': text,
-        }
-        return self._translate('Translate', params, lang_from, lang_to,
+        body = [{
+            'text' : text
+        }]
+        response = self._translate('translate', body, lang_from, lang_to,
                                contenttype, category)
+        if 'error' in response:
+            raise ArgumentOutOfRangeException(response['error']['message'])
+        else:
+            return response[0]['translations'][0]['text']
 
     def translate_array(self, texts=[], lang_from=None, lang_to=None,
                         contenttype='text/plain', category='general'):
-        params = {
-            'texts': json.dumps(texts),
-        }
-        return self._translate('TranslateArray', params, lang_from, lang_to,
+        body = [
+            {'text' : text} for text in texts
+        ]
+        return self._translate('translate', body, lang_from, lang_to,
                                contenttype, category)
 
     def translate_array2(self, texts=[], lang_from=None, lang_to=None,
                         contenttype='text/plain', category='general'):
-        params = {
-            'texts': json.dumps(texts),
-        }
-        return self._translate('TranslateArray2', params, lang_from, lang_to,
-                               contenttype, category)
-
-    def get_translations(self, text, lang_from, lang_to, max_n=10, contenttype='text/plain', category='general',
-                         url=None, user=None, state=None):
-        options = {
-            'Category': category,
-            'ContentType': contenttype,
-        }
-        if url:
-            options['Uri'] = url
-        if user:
-            options['User'] = user
-        if state:
-            options['State'] = state
-        params = {
-            'text': text,
-            'to': lang_to,
-            'from': lang_from,
-            'maxTranslations': max_n,
-            'options': json.dumps(options)
-        }
-        return self.make_request('GetTranslations', params)
+        body = [
+            {'text' : text} for text in texts
+        ]
+        return self._translate('translate', body, lang_from, lang_to,
+                               contenttype, category, includeAlignment = True)
 
     def break_sentences(self, text, lang):
         if len(text) > 10000:
             raise ValueError('The text maximum length is 10000 characters')
         params = {
-            'text': text,
             'language': lang,
         }
-        lengths = self.make_request('BreakSentences', params)
+        body = [
+            {'Text': text}
```

Comment:
Why here we use uppercase `'Text'` and lowercase above?

Reply:
I reviewed official docs and changed to 'Text' everywhere to keep as the official docs.